src/chatbot/run_chatbot.py

```diff
@@ -23,20 +25,31 @@ def modify_response(response: str) -> str:
     return response
 
 
-def get_best_response(responses_list: list[str]) -> str:
+def get_best_response(responses_list: List[str]) -> str:
```

Comment:
@AdrianUrbanski mentioned that `list[str]` should work starting from Python 3.10, maybe it would ba a good idea to decide which Python version we are going to be using?